server/publications/ownUser/emitter.js

```diff
@@ -0,0 +1,27 @@
+import { Notifications } from '../../../app/notifications';
+import { Users } from '../../../app/models';
+import { getDefaultUserFields } from '../../../app/utils/server/functions/getDefaultUserFields';
+
+Users.on('change', ({ clientAction, id, data }) => {
+	switch (clientAction) {
+		case 'inserted':
+			data = Users.findOneById(id, { fields: getDefaultUserFields() });
+			break;
+
+		case 'updated':
+			// Override data cuz we do not publish all fields
+			data = Users.findOneById(id, { fields: getDefaultUserFields() });
+			break;
+
+		case 'removed':
+			data = Users.findOneById(id, { fields: getDefaultUserFields() });
+			break;
+	}
+	// Notifications.streamUser.__emit(data._id, clientAction, data);
+	Notifications.notifyUserInThisInstance(
+		id,
+		'ownUser-changed',
```

Comment:
This seems to be a security issue, anyone can subscribe to any user id and receives private data.

Reply:
@bhardwajaditya

Reply:
Should stream to a particular user be used rather than notifying all users subscribed to it?

Reply:
Changed it to stream to a particular user only